examples/heatmap_2d_subplots.py

```diff
@@ -13,28 +11,14 @@
     "VISam": 1.0,
 }
 
-# Create a list of scenes to plot
-# Note: it's important to keep reference to the scenes to avoid a
-# segmentation fault
-scenes = []
```

Comment:
maybe still worth keeping this example too, for advanced users (like when users want more control over figure size, or more than 25 subplots... or something else) to have more flexibility?